.github/workflows/workflow.yml

```diff
@@ -0,0 +1,26 @@
+on:
```

Comment:
Can we add a workflow name? Also, can this be changed to:
```yaml
on:
  create:
  push:
  pull_request:
    types: [opened, synchronize]
```
As those triggeres seem to work best for other CIs and best for `notify-irc` if we add that.

Reply:
We can